From patchwork Fri Sep  7 05:59:43 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Tianyu Lan <Tianyu.Lan@microsoft.com>
X-Patchwork-Id: 10591661
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id C3297920
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri,  7 Sep 2018 06:00:12 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id AA02C2AF97
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri,  7 Sep 2018 06:00:12 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 9DD242AE92; Fri,  7 Sep 2018 06:00:12 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-8.0 required=2.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham
	version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 4D85B2AF97
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri,  7 Sep 2018 06:00:11 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1725954AbeIGKjJ (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Fri, 7 Sep 2018 06:39:09 -0400
Received: from mail-pu1apc01on0108.outbound.protection.outlook.com
 ([104.47.126.108]:45036
        "EHLO APC01-PU1-obe.outbound.protection.outlook.com"
        rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org with ESMTP
        id S1725843AbeIGKjJ (ORCPT <rfc822;kvm@vger.kernel.org>);
        Fri, 7 Sep 2018 06:39:09 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=selector1;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=hoUSY3ZXKpFzhjY/Zgw6Yp4rBqZUpVNIAaAZC1ykPIQ=;
 b=fFq/zMWgraS8ZfanknRh6+QjWwSWwEWZb1ZCgl+pHMYTBgkRwzKJ7OnQLCH4LJEeOz0L7lmk13Csgi5ONT7KH3gBIvtI6snWyHHLO5EzsX4hp6eQgFT0oR+TRTD62Kq3S1+MqG/sse7jCB0Qx7j7xcruYCaDAtGEE5h+6GX5w+U=
Received: from HK0P153MB0129.APCP153.PROD.OUTLOOK.COM (52.133.156.18) by
 HK0P153MB0145.APCP153.PROD.OUTLOOK.COM (52.133.156.22) with Microsoft SMTP
 Server (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id
 15.20.1143.1; Fri, 7 Sep 2018 05:59:43 +0000
Received: from HK0P153MB0129.APCP153.PROD.OUTLOOK.COM
 ([fe80::194c:c2f9:171:1890]) by HK0P153MB0129.APCP153.PROD.OUTLOOK.COM
 ([fe80::194c:c2f9:171:1890%5]) with mapi id 15.20.1143.008; Fri, 7 Sep 2018
 05:59:43 +0000
From: Tianyu Lan <Tianyu.Lan@microsoft.com>
CC: Tianyu Lan <Tianyu.Lan@microsoft.com>,
        "pbonzini@redhat.com" <pbonzini@redhat.com>,
        "rkrcmar@redhat.com" <rkrcmar@redhat.com>,
        "tglx@linutronix.de" <tglx@linutronix.de>,
        "mingo@redhat.com" <mingo@redhat.com>,
        "hpa@zytor.com" <hpa@zytor.com>, "x86@kernel.org" <x86@kernel.org>,
        "kvm@vger.kernel.org" <kvm@vger.kernel.org>,
        "linux-kernel@vger.kernel.org" <linux-kernel@vger.kernel.org>
Subject: [Resend PATCH] KVM/MMU: Fix comment in
 walk_shadow_page_lockless_end()
Thread-Topic: [Resend PATCH] KVM/MMU: Fix comment in
 walk_shadow_page_lockless_end()
Thread-Index: AQHURm/4ESWgVyRtAkemDFZ3IMKAqg==
Date: Fri, 7 Sep 2018 05:59:43 +0000
Message-ID: <20180907055923.38920-1-Tianyu.Lan@microsoft.com>
References: <20180907054446.37674-1-Tianyu.Lan@microsoft.com>
In-Reply-To: <20180907054446.37674-1-Tianyu.Lan@microsoft.com>
Accept-Language: en-US
Content-Language: en-US
X-MS-Has-Attach: 
X-MS-TNEF-Correlator: 
authentication-results: spf=none (sender IP is )
 smtp.mailfrom=Tianyu.Lan@microsoft.com;
x-originating-ip: [52.168.54.252]
x-ms-publictraffictype: Email
x-microsoft-exchange-diagnostics: 
 1;HK0P153MB0145;6:WEZM61Id/TUMOtXI/DSX1U8hU2353dxdbANKC1up5/iQ8DnFIWgfoq4to+A/WEgSQOlWBcqUf/JAxiqUULYiQco67bk28biuhMS0OHciwmn2RthDylIyVjGzNIvuDZ11SecdykoBJgmYxy5e3q3k36fbb8xNUU7DH6RD3bDbgJzsOLfC4NZ4FdGw4DCnbTyjQTIvUJYSl51AB80NacPnqaWd00GLRUL8aTnLEY0c81keZ9QBTT8fQbPHUM82jSX/50DLCrKE6hsWN6UUr4/+mTFDvdpuCpCV+N+46ZgIq4CZvkqI3OsCxCiAGLFiOElP18GlZDTEwjg+TZ2+3BP+aZvSuXlufmhdtPtQxYZiUAkdxY6kHeoecWAVpHOdHE+g6Gc+WL5drAvv2RxAjmNQoHEIG2nN5KWd41TiX/iolgdLdSe7AjnzkUnJHBnyNWnbe/6zMEup79ruWkJT2Cpb1A==;5:wRg9TJ5IrEaAI5eTgLdkIAI8s9AVVDIgeKt0wxxzA3eprvk8ariwcE+fNjs4+f+CsSV4+4mItszSAzgn29PgYlts+6PnajVyQ5lEe6C7ojVpMBhIG01GrGgsWCqTavL4i9SGzoQ2tF/lMnBehvXIkiFb+WzPJThn8w2hSrXRSqA=;7:g86XS3E4cFLj7PB8di116+NXE7QYz+vR8744oW7YqRd1LBkbQ+6KX0wTU/EAbBQxsSKCEstkJU7uNSQfxQC29hFm9HRdh10kqxQ3nnawPpznLzopd0PmLPdxPeJNU4peRLsvxENr39ci503Xlcx6sOmMzgdUJM7bGoLuKeSH8LRHogmwGbD5esNgeFfeGtTObJq6rZdqyQFpnRot/8Q9x9lHqtp/H87WI7O3TxRS4tdRM9spUzRcPXnmBCsk9qyW
x-ms-office365-filtering-correlation-id: 29fa43e9-2482-44f0-c154-08d614871baa
x-ms-office365-filtering-ht: Tenant
x-microsoft-antispam: 
 BCL:0;PCL:0;RULEID:(7020095)(4652040)(8989137)(5600074)(711020)(4618075)(2017052603328)(7193020);SRVR:HK0P153MB0145;
x-ms-traffictypediagnostic: HK0P153MB0145:
x-microsoft-antispam-prvs: 
 <HK0P153MB0145F5B04786661277F4734092000@HK0P153MB0145.APCP153.PROD.OUTLOOK.COM>
x-exchange-antispam-report-test: UriScan:(28532068793085)(89211679590171);
x-ms-exchange-senderadcheck: 1
x-exchange-antispam-report-cfa-test: 
 BCL:0;PCL:0;RULEID:(8211001083)(6040522)(2401047)(5005006)(8121501046)(3231311)(944501410)(52105095)(3002001)(10201501046)(93006095)(93001095)(6055026)(149027)(150027)(6041310)(20161123558120)(20161123560045)(201703131423095)(201702281528075)(20161123555045)(201703061421075)(201703061406153)(20161123564045)(20161123562045)(201708071742011)(7699050);SRVR:HK0P153MB0145;BCL:0;PCL:0;RULEID:;SRVR:HK0P153MB0145;
x-forefront-prvs: 07880C4932
x-forefront-antispam-report: 
 SFV:NSPM;SFS:(10019020)(366004)(39860400002)(376002)(346002)(136003)(396003)(189003)(199004)(81166006)(66066001)(102836004)(54906003)(8936002)(2906002)(6506007)(76176011)(68736007)(81156014)(7736002)(305945005)(99286004)(486006)(5660300001)(8676002)(2900100001)(476003)(11346002)(478600001)(6486002)(22452003)(97736004)(3846002)(6116002)(5250100002)(446003)(10290500003)(10090500001)(14454004)(53936002)(6512007)(256004)(14444005)(36756003)(86612001)(316002)(109986005)(105586002)(86362001)(6436002)(72206003)(2616005)(106356001)(25786009)(26005)(4326008)(1671002)(59246006)(1076002);DIR:OUT;SFP:1102;SCL:1;SRVR:HK0P153MB0145;H:HK0P153MB0129.APCP153.PROD.OUTLOOK.COM;FPR:;SPF:None;LANG:en;PTR:InfoNoRecords;MX:1;A:1;
received-spf: None (protection.outlook.com: microsoft.com does not designate
 permitted sender hosts)
x-microsoft-antispam-message-info: 
 pKWQSuF94uYVZEXzJromDyuvKfJvQGjOBoLgZK02H/8pJikrddZDXePQwzdEAf7UgZFBAb7LxBDlQEFf+aR1V++XjQx75BasbYFDFyLGPMwijODZp2RO7sbPDgpuHKd6GHMfj6GflHHhGqKeZlcVBtGvIcYc7nIuqK+3oR51AGW+AD21T2J8wo8UKmqpdYr9+KaIMGlfo8r3H8dbbExRd+1pzO1LED4jMde52ZCgtfP7outw5aC4cy17LUxm7LY1V2/nXZjSYFfKYmCQRE6NcdcO7EPX0fL2WfjXXiUAfabl/1Ncw4pznq07tAdYEpHHY7yDlOJ86KjodbsESdfJdhxZLQdk24JdJ8aVTfBmbc8=
spamdiagnosticoutput: 1:99
spamdiagnosticmetadata: NSPM
Content-Type: text/plain; charset="iso-8859-1"
MIME-Version: 1.0
X-OriginatorOrg: microsoft.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 29fa43e9-2482-44f0-c154-08d614871baa
X-MS-Exchange-CrossTenant-originalarrivaltime: 07 Sep 2018 05:59:43.2926
 (UTC)
X-MS-Exchange-CrossTenant-fromentityheader: Hosted
X-MS-Exchange-CrossTenant-id: 72f988bf-86f1-41af-91ab-2d7cd011db47
X-MS-Exchange-Transport-CrossTenantHeadersStamped: HK0P153MB0145
To: unlisted-recipients:; (no To-header on input)
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

kvm_commit_zap_page() has been renamed to kvm_mmu_commit_zap_page()
This patch is to fix the commit.

Signed-off-by: Lan Tianyu <Tianyu.Lan@microsoft.com>
---
Sorry for the broken title and resend.

 arch/x86/kvm/mmu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/kvm/mmu.c b/arch/x86/kvm/mmu.c
index 7ccd29b95746..648b839a349d 100644
--- a/arch/x86/kvm/mmu.c
+++ b/arch/x86/kvm/mmu.c
@@ -947,7 +947,7 @@ static void walk_shadow_page_lockless_end(struct kvm_vcpu *vcpu)
 {
 	/*
 	 * Make sure the write to vcpu->mode is not reordered in front of
-	 * reads to sptes.  If it does, kvm_commit_zap_page() can see us
+	 * reads to sptes.  If it does, kvm_mmu_commit_zap_page() can see us
 	 * OUTSIDE_GUEST_MODE and proceed to free the shadow page table.
 	 */
 	smp_store_release(&vcpu->mode, OUTSIDE_GUEST_MODE);
